report/2.Analysis/EmbeddedPlatforms.tex

```diff
@@ -0,0 +1,76 @@
+% !TeX root = ../main.tex
+% This section is meant to introduce single-board computers that are often used for introducing programmers to beginners.
+% The section itself might not be used a whole lot during the design and implementation, as these are not the arduino platform.
+% Still, it is good to give examples of implementations.
+\section{Embedded platform}
+The goal of the 5th semester, is to develop an embedded system.
+The following sections examines a few different embedded platforms.
+
+\subsection{Lego Mindstorms}
+Lego Mindstorms is a programmable computer kit, made by Lego.
+The platform is based around a central control computer, called the Intelligent Brick.
+It also includes a variety of sensors, motors and connection cables.
+
+The Lego Mindstorm platform, comes with a graphical drag and drop interface.
+However, various different compilers for many different languages exist.
+As such the choice between different generations, does not come down to language preferences.
+
+On the 5th semester of Software, each group is supplied with a lego kit, consisting of both a Lego Mindstorms NXT 2.0 kit and a Lego Mindstorms EV3 kit, aswell as an assortment of sensors, motors and Legos.
```

Comment:
jeg er stadig ikke helt vild med "of software".. det er ikke helt det den hedder :P

Reply:
Har rettet det, men jo det er præcis det den hedder. Man kan lave argumentet at det er bacheloren i software. men jeg har fjernet det